app/src/main/java/pl/pola_app/ui/activity/MainActivity.java

```diff
@@ -91,7 +80,11 @@ public void onBackPressed() {
     @Subscribe
     public void productItemClicked(ProductItemClickedEvent event) {
         if(event.product.company == null && event.product.report.equals(getResources().getString(R.string.ask_for_company_property_name))) {
```

Comment:
ok, it shouldn't happen. We handle null case in onClickListener (can you verify?)